resources/views/profile/comments.blade.php

```diff
@@ -0,0 +1,11 @@
+@extends('layouts.main_layout')
+
+@section('head')
+    <title>{{$username}}s Comments</title>
+@stop
+
+@section('mainBody')
+    <div>
+	<p> Comments </p>
```

Comment:
You can add all a users comments by doing:
[usercomments.txt](https://github.com/uidaho/squireproject/files/249082/usercomments.txt)

You just need to have access the user of the comments you want to show and have the user object be $user